site/docs/v1/tech/apis/_import-users-request-body.adoc

```diff
@@ -163,7 +163,45 @@ When an API key is used that is locked to a Tenant or the tenant `X-FusionAuth-T
 [field]#users``[x]``.timezone# [type]#[String]# [optional]#Optional#::
 The User's preferred timezone. The string must be in an https://www.iana.org/time-zones[IANA] time zone format.
 
-[field]#users``[x]``.twoFactorDelivery# [type]#[String]# [optional]#Optional# [default]#defaults to `None`#::
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.algorithm# [type]#[TOTPAlgorithm]# [optional]#Optional#::
+The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.codeLength# [type]#[Integer]# [optional]#Optional#::
+The length of code generated by the TOTP. With the current implementation, this will always be `6`.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.timeStep# [type]#[Integer]# [optional]#Optional#::
+The time-step size in seconds. With the current implementation, this will always be 30.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.email# [type]#[String]# [optional]#Optional#::
+The value of the email address for this method. Only present if [field]#user.twoFactor.methods``[x]``.method# is `email`.
+
+[field]#users``[x]``.twoFactor.methods``[x]``.id# [type]#[UUID]# [optional]#Optional#::
```

Comment:
This is not a UUID. This is a string. Either that or `methodId` is incorrect here: https://fusionauth.io/docs/v1/tech/apis/two-factor/#disable-multi-factor

Reply:
Yep yep, its a String